### DISCUSSION TITLE
adding a public folder to serve *.txt files directly

### DISCUSSION THREAD

**Discussion by benlutz** [necolas/react-native-web#2454] [Help]
Hey there,

I'm currently trying to add the Apple Pay Verification from stripe and they instructed me to put it the folder `/.well-known/...`.
How would be able to create and serve this folder on my react-native-web-Application? I am hosting my app on amplify and am looking for a way to basically implement a `public`-folder.

Thank you so much in advance!

**Comment by necolas** [comment]
Best to google for help or ask the amplify project, as react-native-web is just a UI component library for react-dom and doesn't have any role to play in hosting or serving files.
